bot/exts/utilities/wtf_python.py

```diff
@@ -79,14 +79,26 @@ def fuzzy_match_header(self, query: str) -> Optional[str]:
         return match if certainty > MINIMUM_CERTAINTY else None
 
     @commands.command(aliases=("wtf", "WTF"))
-    async def wtf_python(self, ctx: commands.Context, *, query: str) -> None:
+    async def wtf_python(self, ctx: commands.Context, *, query: str = None) -> None:
```

Comment:
Pretty trivial but let's update the type hint here
```suggestion
    async def wtf_python(self, ctx: commands.Context, *, query: Optional[str] = None) -> None:
```
Apart from that, LGTM 👍

Reply:
Fixed with [31152da](https://github.com/python-discord/sir-lancebot/pull/939/commits/31152dae2702203aa0ba3b37baaa3a8fea3ff86f)